can-bus/json/INVR/INVR_alerts.json

```
@@ -1,4 +1,3 @@
 {
-    "msg_id": 99,
-    "alerts": []
+    
 }
```

nit: missing blank line
```suggestion
}

```